lib/ansible/modules/network/fortios/fortios_dlp_settings.py

```diff
@@ -0,0 +1,271 @@
+        default: false
+    dlp_settings:
+        description:
+            - Designate logical storage for DLP fingerprint database.
+        default: null
+        suboptions:
+            cache-mem-percent:
+                description:
+                    - Maximum percentage of available memory allocated to caching (1 - 15%).
+            chunk-size:
+                description:
+                    - Maximum fingerprint chunk size.  **Changing will flush the entire database**.
+            db-mode:
+                description:
+                    - Behaviour when the maximum size is reached.
+                choices:
+                    - stop-adding
+                    - remove-modified-then-oldest
+                    - remove-oldest
+            size:
+                description:
+                    - Maximum total size of files within the storage (MB).
+            storage-device:
+                description:
+                    - Storage device name. Source system.storage.name.
+'''
+
+EXAMPLES = '''
+- hosts: localhost
+  vars:
+   host: "192.168.122.40"
+   username: "admin"
+   password: ""
+   vdom: "root"
+  tasks:
+  - name: Designate logical storage for DLP fingerprint database.
+    fortios_dlp_settings:
+      host:  "{{ host }}"
+      username: "{{ username }}"
+      password: "{{ password }}"
+      vdom:  "{{ vdom }}"
+      dlp_settings:
+        cache-mem-percent: "3"
+        chunk-size: "4"
+        db-mode: "stop-adding"
+        size: "6"
+        storage-device: "<your_own_value> (source system.storage.name)"
+'''
+
+RETURN = '''
+build:
+  description: Build number of the fortigate image
+  returned: always
+  type: str
+  sample: '1547'
+http_method:
+  description: Last method used to provision the content into FortiGate
+  returned: always
+  type: str
+  sample: 'PUT'
+http_status:
+  description: Last result given by FortiGate on last operation applied
+  returned: always
+  type: str
+  sample: "200"
+mkey:
+  description: Master key (id) used in the last call to FortiGate
+  returned: success
+  type: str
+  sample: "id"
+name:
+  description: Name of the table used to fulfill the request
+  returned: always
+  type: str
+  sample: "urlfilter"
+path:
+  description: Path of the table used to fulfill the request
+  returned: always
+  type: str
+  sample: "webfilter"
+revision:
+  description: Internal revision number
+  returned: always
+  type: str
+  sample: "17.0.2.10658"
+serial:
+  description: Serial number of the unit
+  returned: always
+  type: str
+  sample: "FGVMEVYYQT3AB5352"
+status:
+  description: Indication of the operation's result
+  returned: always
+  type: str
+  sample: "success"
+vdom:
+  description: Virtual domain used
+  returned: always
+  type: str
+  sample: "root"
+version:
+  description: Version of the FortiGate
+  returned: always
+  type: str
+  sample: "v5.6.3"
+
+'''
+
+from ansible.module_utils.basic import AnsibleModule
+
+fos = None
+
+
+def login(data):
```

Comment:
Might want to move this function to `module_utils` so that other FortiOS modules can use it.